e2e-tests/trade-finance-manager/cypress/integration/journeys/case/deal-MIA.spec.js

```diff
@@ -64,6 +64,8 @@ context('User can view a case deal', () => {
       pages.caseDealPage.bankSecuritySectionHeading().contains('Bank security');
       pages.caseDealPage.bankSecuritySubHeading().contains('General bank security for this exporter');
       pages.caseDealPage.bankSecurityText().contains('Mock security details');
+      pages.caseDealPage.bankSecurityFacilitySubHeading().contains('Specific bank security for the facilities listed in this application');
+      pages.caseDealPage.bankSecurityFacilityText().contains('Mock facility details');
```

Comment:
If it should only display/exist for GEF deal this test is misleading as it's using a mock BSS deal